content/posts/0006-build-plan-buildpack.md

```diff
@@ -0,0 +1,94 @@
+---
+title: What is the Build Plan Buildpack and why should I care?
+date: "2021-03-05"
+slug: build-plan-buildpack-exploration
+author: feckhardt
+---
+The [Build Plan buildpack](https://github.com/paketo-community/build-plan)
+gives you very precise control over what buildpacks run during a build and what
+dependencies end up in the final image. Let's talk about how it works and
+why/when you would utilize this kind of functionality.
+
+### What is the Build Plan?
+
+The Build Plan is a part of the Cloud Native Buildpacks (CNB) specification
+that is the format for how buildpacks communicate during the [detect phase](https://paketo.io/docs/buildpacks/#detect-phase) of
+the lifecycle. Buildpacks write out a Build Plan that outlines the
+dependency that the buildpack will either provide or require during its build
+phase. Buildpacks may both require and provide any number of dependencies. For
+detection to pass a buildpacks must have all of the dependencies it requires,
+provided by a buildpack that comes before it in the build order and it needs to
+have all of the dependencies that it provides, required by buildpacks that come
+later in the build order. If either of these conditions is not satisfied
+detection will fail for that buildpack. A more in-depth breakdown of the Build
+Plan can be found in the [CNB
+Spec](https://github.com/buildpacks/spec/blob/main/buildpack.md).
+
+### How does the Build Plan Buildpack work?
+
+The Build Plan Buildpack works by adding a `plan.toml`, which mirrors the
+[requires portion of Build
+Plan](https://github.com/buildpacks/spec/blob/main/buildpack.md#build-plan-toml),
+to the root of your app. The buildpack will read the `plan.toml` and make all
```

Comment:
This makes me think that the buildpack is the one that adds the `plan.toml` to the root directory. Maybe you could clarify your wording here?